src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -1049,29 +1049,55 @@ void ReduceAxesBackwardUseInOut(const nnvm::NodeAttrs& attrs,
   ReduceAxesBackwardUseInOutImpl<xpu, OP, normalize>(ctx, small, inputs, req, outputs);
 }
 
+namespace {  // unnamed namespace to keep scope of the struct within the file
+struct shape_and_stride {
```

Comment:
Please use CamelCase naming convention for struct. See this: https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
Will do. But its confusing that when struct is used as kernel then its using `_` but when just as struct it is camel cased in our code

Reply:
I agree with @apeforest in terms of the CamelCase standard.